hack/e2e.sh

```diff
@@ -30,6 +30,8 @@ METERING_SHORT_TESTS="${METERING_SHORT_TESTS:=false}"
 METERING_REPO_PATH="${METERING_REPO_PATH:=${ROOT_DIR}}"
 METERING_REPO_VERSION="$(basename $(find $MANIFESTS_BASE_DIR -type d | sort -r | head -n 1))"
 METERING_RUN_TESTS_LOCALLY="${METERING_RUN_TESTS_LOCALLY:=false}"
+METERING_OLM_SUBSCRIPTION_CHANNEL="${METERING_OLM_SUBSCRIPTION_CHANNEL:="4.6"}"
+METERING_ANSIBLE_OPERATOR_IMAGE_REGISTRY="${METERING_ANSIBLE_OPERATOR_IMAGE_REGISTRY:=""}"
```

Comment:
Something that should be noted is that we don't delete the registry resources that get created (deployment, service, CatalogSource) if you kill the `make e2e` process, but we do defer the deletion of those resources as long as the testing suite wasn't run in developer mode.